my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './my_enumarable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Kindly in each method you need to implement its login not directly use each() method inside it. Note you can use the yield keyword to do that.